docker-compose-deploy.yaml

```diff
@@ -0,0 +1,33 @@
+version: "3.8"
+# this is if you want to build from our docker repo
+# use the command: docker-compose -f docker-compose-deploy.yaml up
+services:
+  backend:
+    restart: always
+    container_name: backend
+    command: bash -c "cd lead && python manage.py runserver 0.0.0.0:8000"
+    image: judgerussell/track2backend
+    ports:
+      - "8000:8000"
+    networks:
+      default-net
+
+  frontend:
+    restart: always
+    command : npm start
+    container_name: frontend
+    image: judgerussell/track2frontend
+    ports:
+      - "3000:3000"
+    stdin_open: true
+    depends_on:
+      - backend
+    networks:
+      default-net
+
```

Comment:
services.backend.networks contains an invalid type, it should be an array, or an object
services.frontend.networks contains an invalid type, it should be an array, or an object

Think you're missing a dash, also maybe keep naming of services consistent (back vs backend, front vs frontend)